modules/boltlib/lib/puppet/functions/run_task.rb

```diff
@@ -24,10 +26,25 @@
     block_param
   end
 
+  def log_summary(mode, object, node_count, fail_count)
+    format("Ran %s %s on %d node%s with %d failure%s",
+           mode,
+           object,
+           node_count,
+           node_count == 1 ? '' : 's',
+           fail_count,
+           fail_count == 1 ? '' : 's')
+  end
+
   def run_task(task_name, targets, task_args = nil)
-    Bolt::ExecutionResult.from_bolt(
+    logger = Logger.new(STDOUT)
```

Comment:
We should never log to stdout. This needs access to a configured bolt logger

Reply:
I'm not sure how to do that. Is there a way to pass a configured logger to that puppet function, or do I need to read the config in this file?